bitnami/airflow/Chart.yaml

```
@@ -41,4 +41,4 @@ maintainers:
 name: airflow
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/airflow
-version: 25.0.4
+version: 25.0.5
```

As you are adding new parameters, please bump the chart to a new minor version

```suggestion
version: 25.1.0
```